### PR TITLE
Add irssi official image docs!

### PR DESCRIPTION
**Pull request by jfrazelle**

### CODE REVIEW

File: irssi/content.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# What is irssi?
+
+Irssi is a terminal based IRC client for UNIX systems. It also supports SILC
+and ICB protocols via plugins. Some people refer to it as 'the client of the
+future'.
+
+> [irssi.org](http://irssi.org)
+
+![logo](https://raw.githubusercontent.com/docker-library/docs/master/irssi/logo.png)
```
Comment (tianon) [inline irssi/content.md]: This should be `%%LOGO%%`.

File: irssi/content.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+configuring irssi for your needs.
+
+Be sure to also checkout the [awesome
+scripts](https://github.com/irssi/scripts.irssi.org) you can download to
+customize your irssi configuration.
+
+## Create a `Dockerfile`
+
+    FROM irssi:latest
+    COPY .irssi/ /home/user/.irssi
```
Comment (tianon) [inline irssi/content.md]: hmm, since our `.irssi` is a `VOLUME`, will this actually work?

Comment (jfrazelle) [inline irssi/content.md]: whoops sorry forgot to use my brain when copying from haproxy, I dont think we need this even

Comment (tianon) [inline irssi/content.md]: +++

**Comment by tianon** [comment]
LGTM

**Comment by tianon** [comment]
Still LGTM

**Comment by jfrazelle** [comment]
updated :D

**Comment by tianon** [comment]
Oh, do you think we should mention `/etc/localtime` too?

**Comment by tianon** [comment]
Since this is one of the few cases where it's actually relevant and `/etc/localtime` being bind-mounted is probably exactly what users want. :+1:

**Comment by jfrazelle** [comment]
Omg yes I use that in my alias :) I'll add it

On Friday, January 23, 2015, Tianon Gravi <notifications@github.com> wrote:

> Since this is one of the few cases where it's actually relevant and
> /etc/localtime being bind-mounted is probably exactly what users want. [image:
> :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker-library/docs/pull/145#issuecomment-71258336>.
>


-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>

**Comment by tianon** [comment]
Right?  I was looking at my own run script and realized that's probably
something most users want. :)

(probably with :ro, to be safe)

**Comment by tianon** [comment]
Oh, you need to add `irssi` in `update.sh` too, with a pointer to your repo.

**Comment by jfrazelle** [comment]
done :)

On Fri, Jan 23, 2015 at 12:37 PM, Tianon Gravi <notifications@github.com>
wrote:

> Oh, you need to add irssi in update.sh too, with a pointer to your repo.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker-library/docs/pull/145#issuecomment-71261495>.
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>

**Comment by yosifkit** [comment]
LGTM

File: update.sh [hunk 3]
```diff
@@ -33,6 +33,7 @@ declare -A otherRepos=(
 	[hipache]='https://github.com/dotcloud/hipache'
 	[hylang]='https://github.com/hylang/hy'
 	[iojs]='https://github.com/iojs/docker-iojs'
+	[irssi]='https://github.com/jfrazelle/irssi'
```
Comment (tianon) [inline update.sh]: Glad to see you figured out your alphabet. :+1:

**Comment by tianon** [comment]
LGTM

**Pull request merged**